### PR TITLE
LinkedIn Ads source documentation

### PR DESCRIPTION
**Pull request by mohan-h**
Initial commit

### CODE REVIEW

File: help/sources/tutorials/api/create/advertising/linkedin-ads.md [hunk 1]
```diff
@@ -0,0 +1,1060 @@
...
+| --- | --- |
+| `name` | The name of your source connection. Ensure that the name of your source connection is descriptive as you can use this to look up information on your source connection. |
+| `description` | An optional value that you can include to provide more information on your source connection. |
+| `baseConnectionId` | The base connection ID of LinkedIn Ads. This ID was generated in an earlier step. |
+| `connectionSpec.id` | The connection specification ID that corresponds to your source. |
+| `data.format` | The format of the LinkedIn Ads data that you want to ingest. Currently, the only supported data format is `json`. |
+| `accountIds` | Your LinkedIn account IDs |
+| `campaignGroupIds` | Your LinkedIn Campaign Group IDs. |
+| `campaignIds` | Your LinkedIn Campaign IDs. |
+| `pivot` | One of `ACCOUNT`, `CAMPAIGN GROUP`, `CAMPAIGN` or `CREATIVE` |
```
Comment (carlojavier) [inline help/sources/tutorials/api/create/advertising/linkedin-ads.md]: Can we add a brief definition or description to what `pivot` is?

File: help/sources/tutorials/api/create/advertising/linkedin-ads.md [hunk 2]
```diff
@@ -0,0 +1,1060 @@
...
+hide: true
+hidefromtoc: true
+---
+# (Beta) Create a source connection and dataflow for [!DNL LinkedIn Ads] using the Flow Service API
+
+>[!NOTE]
+>
+>The [!DNL LinkedIn Ads] source is in beta. See the [sources overview](../../../../home.md#terms-and-conditions) for more information on using beta-labelled sources.
+
+[[!DNL LinkedIn Ads]](https://business.linkedin.com/marketing-solutions/ads) is a marketing solution by LinkedIn to reach over 830 million professionals on the world's largest professional network. The LinkedIn Reporting & ROI APIs provide key insights on performance such as clicks, impressions, and ad spend, and demographics information such as metrics by demographic values at the account, campaign, and creative levels.
```
Comment (carlojavier) [inline help/sources/tutorials/api/create/advertising/linkedin-ads.md]: - Let's rephrase the first sentence to: "...is a marketing solution by LinkedIn that can reach over 830..."
- And a slight restructuring to the second sentence: "...key insights on performance such as clicks, impressions, and ad spend, as well as demographic information like metrics by demographic values at the account, campaign, and creative levels."

File: help/sources/tutorials/api/create/advertising/linkedin-ads.md [hunk 3]
```diff
@@ -0,0 +1,1060 @@
...
+
+This Adobe Experience Platform [source](https://experienceleague.adobe.com/docs/experience-platform/sources/home.html?lang=en) leverages the [LinkedIn Reporting & ROI APIs](https://docs.microsoft.com/en-us/linkedin/marketing/integrations/ads-reporting/ads-reporting) to retrieve your LinkedIn campaign data, along with all performance and metrics into Experience Platform allowing you to execute analytics. The data is returned from a specified date for the default 6 month retention on a daily basis. Refer to [guardrails](#guardrails) for details.
```
Comment (carlojavier) [inline help/sources/tutorials/api/create/advertising/linkedin-ads.md]: - Some restructuring here: "...to retrieve your LinkedIn campaign data, along with all performance and metrics. You can then bring that data to Experience Platform, where you can execute analytics."
- "The data is returned from a specified date with a default of six month retention period for daily data." (Let me know if this rephrasing still works for the meaning.)

File: help/sources/tutorials/api/create/advertising/linkedin-ads.md [hunk 4]
```diff
@@ -0,0 +1,1060 @@
...
+
+In order to connect [!DNL LinkedIn Ads] to Platform, you must provide values for the following connection properties:
+
+| Credential | Description | Example |
+| --- | --- | --- |
+| Host | LinkedIn API endpoint. | `https://api.linkedin.com` |
+| Access Token | Assuming you have a LinkedIn Developer Application follow this [!DNL LinkedIn] tutorial [Authorization Code Flow (3-legged OAuth)](https://docs.microsoft.com/en-us/linkedin/shared/authentication/authorization-code-flow) to generate the access token. | `AQV4FClIuE2BbDkQeDatkhMFs................qNAmFmlcr9A` |
+| Account IDs | You can find guidance to obtain the Account ID [here](https://www.linkedin.com/help/lms/answer/a424270). | `508670032` |
+| Campaign Group IDs | Campaign Group IDs can be obtained by navigating to the Campaign Groups page and selecting the desired CIDs. | `611138362` |
+| Campaign IDs | You can find guidance to obtain the Campaign Group ID [here](https://www.linkedin.com/help/lms/answer/a424270). | `1991010101,1992020202` |
```
Comment (carlojavier) [inline help/sources/tutorials/api/create/advertising/linkedin-ads.md]: For the API side, can we wrap the credentials in backticks and write them out in camelcase

File: help/sources/tutorials/api/create/advertising/linkedin-ads.md [hunk 5]
```diff
@@ -0,0 +1,1060 @@
...
+
+### Guardrails {#guardrails}
+
+See the Retention, Restrictions, Delays and Accuracy sections on the [LinkedIn Reporting & ROI APIs](https://docs.microsoft.com/en-us/linkedin/marketing/integrations/ads-reporting/ads-reporting) page. Note, [!DNL LinkedIn Ads] only allows data retrieval from campaigns that are active. The data is returned from a specified date for the default 6 month retention on a daily basis.
```
Comment (carlojavier) [inline help/sources/tutorials/api/create/advertising/linkedin-ads.md]: Let's move the second sentence into an actual note:

>[!NOTE]
>
>LinkedIn Ads only allows data retrieval from active campaigns.The data is returned from a specified date with a default of six month retention period for daily data."

File: help/sources/tutorials/ui/create/advertising/linkedin-ads.md [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+> | Day | 1 |
+> | Hour | 24 |
+
+![Platform UI screenshot for LinkedIn Ads showing scheduling](../../../../images/tutorials/create/linkedin-ads/scheduling.png?lang=en)
+
+## Additional resources
+
+### Validation {#validation}
+
+To validate that you have correctly set up the source; and [!DNL LinkedIn Ads] events are being ingested, follow the steps below:
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/advertising/linkedin-ads.md]: Semi-colon (`;`) is unnecessary here

File: help/sources/tutorials/ui/create/advertising/linkedin-ads.md [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+
+1. In the Platform UI, select View Dataflows besides the [!DNL LinkedIn Ads] card menu on the Catalog page. You can then select [!UIControl Preview dataset] to verify the data that was ingested.
+![Platform UI screenshot for LinkedIn Ads Preview Dataset](../../../../images/tutorials/create/linkedin-ads/preview-dataset.png?lang=en)
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/advertising/linkedin-ads.md:122]: Can the screenshot be updated here? There is a typo on the dataset name "Tset123"

File: help/sources/tutorials/ui/create/advertising/linkedin-ads.md [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+
+>[!NOTE]
+>
+>The [!DNL LinkedIn Ads] source is in beta. See the [sources overview](../../../../home.md#terms-and-conditions) for more information on using beta-labelled sources.
+
+This tutorial provides steps for creating a [!DNL LinkedIn Ads] source connector using the Platform user interface.
+
+## Overview
+
+[[!DNL LinkedIn Ads]](https://business.linkedin.com/marketing-solutions/ads) is a marketing solution by LinkedIn to reach over 830 million professionals on the world's largest professional network. The LinkedIn Reporting & ROI APIs provide key insights on performance such as clicks, impressions, and ad spend, and demographics information such as metrics by demographic values at the account, campaign, and creative levels.
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/advertising/linkedin-ads.md]: Same as API:
Let's rephrase the first sentence to: "...is a marketing solution by LinkedIn that can reach over 830..."
And a slight restructuring to the second sentence: "...key insights on performance such as clicks, impressions, and ad spend, as well as demographic information like metrics by demographic values at the account, campaign, and creative levels."

File: help/sources/tutorials/ui/create/advertising/linkedin-ads.md [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+| Access Token | Assuming you have a LinkedIn Developer Application follow this LinkedIn tutorial [Authorization Code Flow (3-legged OAuth)](https://docs.microsoft.com/en-us/linkedin/shared/authentication/authorization-code-flow) to generate the access token. | `AQV4FClIuE2BbDkQeDatkhMFs................qNAmFmlcr9A` |
+| Account IDs | You can find guidance to obtain the Account ID [here](https://www.linkedin.com/help/lms/answer/a424270). | `508670032` |
+| Campaign Group IDs | Campaign Group IDs can be obtained by navigating to the Campaign Groups page and selecting the desired CIDs. | `611138362` |
+| Campaign IDs | You can find guidance to obtain the Campaign Group ID [here](https://www.linkedin.com/help/lms/answer/a424270). | `1991010101,1992020202` |
+
+### Guardrails {#guardrails}
+
+See the Retention, Restrictions, Delays and Accuracy sections on the [LinkedIn Reporting & ROI APIs](https://docs.microsoft.com/en-us/linkedin/marketing/integrations/ads-reporting/ads-reporting) page. Note, [!DNL LinkedIn Ads] only allows data retrieval from campaigns that are active. The data is returned from a specified date for the default 6 month retention on a daily basis.
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/advertising/linkedin-ads.md]: Same comment as API about moving the note into its own [!NOTE] tag

File: help/sources/tutorials/ui/create/advertising/linkedin-ads.md [hunk 10]
```diff
@@ -0,0 +1,150 @@
...
+> Multiple IDs can be specified by passing comma separated values.
+
+If you fail to provide a value on selecting **[!UICONTROL Select]** you will get a `Failed to parse JSON.` or `Unknown error while processing request.` error message.
+
+![Platform UI screenshot for LinkedIn Ads showing configuration](../../../../images/tutorials/create/linkedin-ads/configuration.png?lang=en)
+
+## Next steps
+
+By following this tutorial, you have established a connection to your [!DNL LinkedIn Ads] account. You can now continue on to the next tutorial and [configure a dataflow to bring data into Platform](/help/sources/ui-tutorials/dataflow/crm.md).
+
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/advertising/linkedin-ads.md:104]: I think we need to preface this particular part about why a user needs to schedule specifically for Day or Hour? Can we add a brief intro here on why this is recommended/required?

File: help/sources/tutorials/ui/create/advertising/linkedin-ads.md [hunk 11]
```diff
@@ -0,0 +1,150 @@
...
+Finally, you must provide as a mandatory input one or more of the field values below:
+| Field | Description |
+| --- | --- |
+| accountIds | Your LinkedIn account IDs. |
+| campaignGroupIds | Your LinkedIn Campaign Group IDs. |
+| campaignIds | Your LinkedIn Campaign IDs. |
+
+> [!NOTE]
+>
+> Multiple IDs can be specified by passing comma separated values.
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/advertising/linkedin-ads.md]: Need to remove space between ">" and the text

File: help/sources/tutorials/ui/create/advertising/linkedin-ads.md [hunk 12]
```diff
@@ -0,0 +1,150 @@
...
+   * CAMPAIGN
+   * CREATIVE
+They will enable you to restrict export to the specified Account IDs, Campaign Group IDs or Campaign IDs. Refer to [LinkedIn Reporting & ROI API Query Parameters](https://docs.microsoft.com/en-us/linkedin/marketing/integrations/ads-reporting/ads-reporting?view=li-lms-2022-07&tabs=http#query-parameters) to understand pivoting.
+
+Finally, you must provide as a mandatory input one or more of the field values below:
+| Field | Description |
+| --- | --- |
+| accountIds | Your LinkedIn account IDs. |
+| campaignGroupIds | Your LinkedIn Campaign Group IDs. |
+| campaignIds | Your LinkedIn Campaign IDs. |
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/advertising/linkedin-ads.md]: let's wrap the entries in the fields column in backticks

**Review by carlojavier — CHANGES_REQUESTED** [review]
@mohan-h looking good, some comments added!

File: help/sources/tutorials/api/create/advertising/linkedin-ads.md (hunk 3 above)
Comment (mohan-h) [reply help/sources/tutorials/api/create/advertising/linkedin-ads.md]: Used _The data is returned from a specified date within the six month default retention period for daily data._

Comment (mohan-h) [reply help/sources/tutorials/api/create/advertising/linkedin-ads.md]: Used _The data is returned from a specified date within the six month default retention period for daily data._

**Comment by carlojavier** [comment]
Closing due to inactivity. The PR is ready on internal, still just waiting on approval from internal on when it can be published.

**Pull request closed without merging**